runtime-modules/content/src/lib.rs

```diff
@@ -1335,6 +1579,11 @@ impl<T: Trait> Module<T> {
 decl_event!(
     pub enum Event<T>
     where
+        Actor = ContentActor<
```

Comment:
Try to always keep the same type name in the events, to avoid need to add type alias names in our javascript types library.
So:
```suggestion
        ContentActor = ContentActor<
```

Reply:
Ok, great https://github.com/Joystream/joystream/pull/2194/commits/1659081ca0a650efb68fe727321a48c2a69cf808